tests/dag_processing/bundles/test_dag_bundle_manager.py

```diff
@@ -51,23 +60,28 @@ def test_bundle_configs_property(envs, expected_names):
     """Test that bundle_configs are read from configuration."""
     bundle_manager = DagBundlesManager()
     with patch.dict(os.environ, envs):
-        names = set(bundle_manager.bundle_configs.keys())
+        bundle_manager.parse_config()
```

Comment:
Should this be private? Doesn't feel like the caller should call parse explicitly.

Reply:
it doesn't really bother me 🤷